umm-spec-lib/src/cmr/umm_spec/migration/version/variable.clj

```diff
@@ -3,13 +3,60 @@
   (:require
    [cmr.umm-spec.migration.version.interface :as interface]))
 
+(defn sample-11->12
+  [x]
+  {:MeasurementConditions (:MeasurementConditions x)
+   :ReportingConditions (:ReportingConditions x)})
+
+(defn measurement-11->12
+  [x]
+  {:MeasurementSource (:MeasurementSource x)
+   :MeasurementName {:MeasurementObject (:MeasurementName x)}})
+
+(defn measurement-12->11
+  [x]
+  {:MeasurementSource (:MeasurementSource x)
+   :MeasurementName (get-in x [:MeasurementName :MeasurementObject])})
+
+(defn dimensions-12->11
+  [dims]
+  (mapv #(dissoc % :Type) dims))
+
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Variable Migration Implementations
 
 (defmethod interface/migrate-umm-version [:variable "1.0" "1.1"]
+  ;; Migrate up to 1.1
   [context v & _]
-  (dissoc v :Services))
```

Comment:
It looks like this change to an old migration was intentional (and something must have been broken before). Do we have a test for the 1.0->1.1 and 1.1->1.0 migrations now?

Reply:
I found a test - ignore.